zombienet_tests/smoke/0002-parachains-upgrade-smoke-test.toml

```diff
@@ -26,11 +26,12 @@ command = "polkadot"
 [[parachains]]
 id = 100
 addToGenesis = true
+cumulus_based = true
 
   [parachains.collator]
   name = "collator01"
   image = "{{COL_IMAGE}}"
-  commandWithArgs = "polkadot-collator --collator --alice --force-authoring --tmp --parachain-id 100 --rpc-cors all --unsafe-rpc-external --rpc-methods unsafe --unsafe-ws-external -- --execution wasm --chain /cfg/rococo-local.json --bootnodes /dns/bootnode/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp"
```

Comment:
Why are those args no longer needed?

Reply:
it's handled by zombienet (some args inferred by cumulus_based):
https://github.com/paritytech/zombienet/blob/465622bc84f9824d8c712cc8aef74f4ee223ab0d/src/cmdGenerator.ts#L40